proto/resource_manager.proto

```diff
@@ -28,6 +28,7 @@ service ResourceManager {
 
 message ListResourceGroupsRequest{
   bool with_ru_stats = 1;
+  uint32 keyspace_id = 2;
```

Comment:
Do we need to provide keyspace name?

Reply:
Keyspace ID is its unique identifier; it seems unnecessary to introduce a name.